invenio_accounts/testutils.py

```diff
@@ -66,6 +66,27 @@ def create_test_user(email='test@test.org',
     return user
 
 
+def create_legacy_user(email='test@test.org', password='123456', **kwargs):
```

Comment:
I don't think we should add this method. The ``create_test_user`` is an API function used by other modules to create a user in a fast and easy way, and we don't want to allow other modules to create users with legacy passwords.